Testing/SystemTests/tests/analysis/VesuvioTesting.py

```diff
@@ -0,0 +1,514 @@
+from mantid.simpleapi import *
```

Comment:
Yes, I think we could add this to a `testing.py` module of the `vesuvio` package. It shouldn't do any harm.